docker-compose-test.yml

```diff
@@ -23,6 +23,174 @@ services:
       - ./docker/keycloak:/opt/keycloak/data/import
     command: start-dev --import-realm
 
+  # -------------------------------------------------------------------- extra attachment svcs
```

Comment:
We should move these changes to the regular compose as these aren't used by the tests